tests/suites/network/network_health.sh

```diff
@@ -6,21 +6,21 @@ run_network_health() {
 	ensure "network-health" "${file}"
 
 	# Deploy some applications for different series.
-	juju deploy ubuntu ubuntu-focal --series focal
-	juju deploy ubuntu ubuntu-jammy --series jammy
+	juju deploy ./tests/suites/network/charms/ubuntu ubuntu-focal --series focal
```

Comment:
These are CI tests and should deploy charms like ubuntu (which is a supported, published charm) from charmhub, not a local directory. If this change is from 2.9, we need to revert it there.